driver/Dockerfile

```diff
@@ -11,7 +11,10 @@ RUN apt-get update && apt-get install -y git wget bash bzip2 zip && \
     rm -r /var/cache/apt/* && \
     wget -qO- https://api.anaconda.org/download/conda-forge/micromamba/0.8.2/linux-64/micromamba-0.8.2-he9b6cbd_0.tar.bz2 | tar -xvj -C /usr/local bin/micromamba && \
     mkdir $MAMBA_ROOT_PREFIX && \
-    micromamba install -y --prefix $MAMBA_ROOT_PREFIX -c cpg -c bioconda -c conda-forge hail=$HAIL_VERSION analysis-runner bokeh selenium phantomjs && \
+    micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
+        -c cpg -c bioconda -c conda-forge \
+        hail=$HAIL_VERSION analysis-runner bokeh selenium phantomjs \
+        r-base=4.0.3 r-essentials r-tidyverse && \
```

Comment:
Is there a particular reason we pin `r-base` to 4.0.3?

Reply:
That was the latest version, it looks like r-base do maintain and update some older versions.